core/domain/collection_domain_test.py

```diff
@@ -641,9 +619,33 @@ class SchemaMigrationUnitTests(test_utils.GenericTestBase):
 tags: []
 title: A title
 """)
+    YAML_CONTENT_V3 = ("""category: A category
+language_code: en
+nodes:
+- acquired_skills:
+  - 0
+  - 1
+  exploration_id: Exp1
+  prerequisite_skills: []
+objective: ''
+schema_version: 2
+skills:
+  0:
```

Comment:
oh, didn't know that. It might be cleaner to store everything as strings, in that case (we'll convert IDs to strings on creation). I think that's a  better idea, and requires less refactoring.